packages/typespec-client-generator-core/src/types.ts

```diff
@@ -1251,7 +1252,8 @@ function updateMultiPartInfo(
   context: TCGCContext,
   type: ModelProperty,
   base: SdkBodyModelPropertyType,
-  operation: Operation
+  operation: Operation,
+  httpOperation: HttpOperation
```

Comment:
`updateMultiPartInfo` is just Internal function and now that we already get `httpOperation`, it is better to pass it to `updateMultiPartInfo` instead of call getHttpOperationCache to get it.